### PR TITLE
Chore/Add PR and Issues template

### PR DESCRIPTION
**Pull request by nathan-vm**
Depends on: https://github.com/Tauffer-Consulting/domino-docs/pull/31

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+## Issues
+
+### Issues guidance links
+
+* [Asking for General Help](https://github.com/Tauffer-Consulting/domino/discussions)
+* [Discussing non-technical topics](https://github.com/Tauffer-Consulting/domino/discussions)
+* [Submitting a Bug Report](https://github.com/Tauffer-Consulting/domino/issues/new/choose)
+
+### Submitting a issue
```
Comment (vinicvaz) [inline CONTRIBUTING.md:24]:
```suggestion
### Submitting an issue
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+The two most important pieces of information we need in order to properly evaluate the report is a description of the behavior you are seeing and a simple test case we can use to recreate the problem on our own. If we cannot recreate the issue, it becomes impossible for us to fix.
+
+Once an issue has been opened, it is common for there to be discussion around it. Some contributors may have differing opinions about the issue, including whether the behavior being seen is a bug or a feature. This discussion is part of the process and should be kept focused, helpful, and professional.
+
+## Pull Requests
+
+A pull request is a proposal to merge a set of changes from one branch into another. In a pull request, collaborators can review and discuss the proposed set of changes before they integrate the changes into the main codebase.
+
+Before open a pull request, be sure you read and agree with our [Developer's Certificate of Origin](#developers-certificate-of-origin).
```
Comment (vinicvaz) [inline CONTRIBUTING.md:36]:
```suggestion
Before opening a pull request, be sure you read and agree with our [Developer's Certificate of Origin](#developers-certificate-of-origin).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+```bash
+git fetch upstream dev
+git rebase dev
+```
+
+This ensures that your working branch has the latest changes from `Tauffer-Consulting/domino` development branch.
+
+### Step 6 - Opening the pull request
+
+From within GitHub, opening a new pull request will present you with a [pull request template][]. Please try to do your best at filling out the details, but feel free to skip parts if you're not sure what to put.
```
Comment (vinicvaz) [inline CONTRIBUTING.md:106]: Missing pull request template link / ref

File: .github/ISSUE_TEMPLATE/2-docs-report.yml
Comment (vinicvaz) [inline .github/ISSUE_TEMPLATE/2-docs-report.yml:1]: Maybe for docs we should put directly in the docs repository and just reference it here, what you think @nathan-vm ?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (vinicvaz) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:1]: Why is this commented?

**Review by vinicvaz — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md
Comment (nathan-vm) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:1]: It's read-only

**Pull request merged**